### PR TITLE
DOCS-862: Update gantry model names

### PR DESCRIPTION
**Pull request by andf-viam**
Update gantry model names, per **#eng-breaking-changes** thread.

- Changes are:
	- `oneaxis` -> `single-axis`
	- `multiaxis` -> `multi-axis`

NOTE: Don't merge until release containing this fix is merged. Estimated: ~June 27

### CODE REVIEW

File: docs/components/gantry/multi-axis.md [hunk 1]
```diff
@@ -202,8 +202,8 @@ Edit and fill in the attributes as applicable.
 {{% /tab %}}
 {{% /tabs %}}
 
-The following attributes are available for `multiaxis` gantries:
+The following attributes are available for `multi-axis` gantries:
 
 | Attribute | Type | Inclusion | Description |
 | --------- | ---- | --------- | ----------- |
-| `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [one-axis](/components/gantry/oneaxis/) gantries that make up the multi-axis gantry. |
+| `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [`single-axis`](/components/gantry/single-axis/) gantries that make up the `multi-axis` gantry. |
```
Comment (JessamyT) [inline docs/components/gantry/multi-axis.md:209]: [opt] outside of diff, but if you want to make this sentence less awk while you're at it, I don't have a good idea but maybe you do!

**Review by JessamyT — APPROVED** [review]
LGTM

File: docs/components/gantry/multi-axis.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
 ---
 title: "Configure a Multi-Axis Gantry"
-linkTitle: "multiaxis"
+linkTitle: "multi-axis"
 weight: 80
 type: "docs"
-description: "Configure a multiaxis gantry."
+description: "Configure a multi-axis gantry."
 images: ["/components/img/components/gantry.svg"]
 tags: ["gantry", "components"]
```
Comment (npentrel) [inline docs/components/gantry/multi-axis.md:8]: please add an alias to the old name for both of these

**Review by npentrel — APPROVED** [review]
LGTM once aliases are added

File: docs/components/gantry/multi-axis.md (hunk 1 above)
Comment (andf-viam) [reply docs/components/gantry/multi-axis.md:209]: Hmm, explored a bit but don't have one either at the moment. Might circle back later.

**Review by npentrel — APPROVED** [review]

**Comment by npentrel** [comment]
ready to merge?

**Comment by andf-viam** [comment]
@npentrel Need guidance then:

The eng chang does appear for [v0.3.0-rc0](https://github.com/viamrobotics/rdk/releases/tag/v0.3.0-rc0):

> [RSDK-3645](https://viam.atlassian.net/browse/RSDK-3645): gantry scope config changes by @martha-johnston in https://github.com/viamrobotics/rdk/pull/2518

Do we merge docs for features when they:
- Make it to an rc i.e. pre-release i.e. `unstable`?
- Only to a GA, i.e. `stable`?

**Comment by npentrel** [comment]
No you're right, we should wait until the ticket is closed. My bad

**Comment by andf-viam** [comment]
v0.3.0 is live, merging this PR!

**Pull request merged**